### ISSUE TITLE
Compliance/Compliance Obligation view - Monetary payments made

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-compliance#98]
#### Description:

Operators can view all payments made towards their obligation, and details about the payment.

[Figma](https://www.figma.com/proto/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=20107-1258&t=D4UaBo6pmGx65d5s-1)
#### Acceptance Criteria:

As an operator, I have the ability to view the details of monetary payments made towards my compliance obligation.

Monetary Payments Made section

- [ ] Section header: Monetary Payments Made
- [ ] A table showing monetary payments made towards the obligation
- [ ] Table header: Payment received date, Payment amount applied, payment method, transaction type, reference number
- [ ] The table header row can be sorted and there's a free-text search box

Data source for the table 

- [ ] eLicensing API

**Note:** Ticket includes front end UI and back end work

Given that I am an external user 
When I am in the Compliance Obligation workflow
Then I can view Monetary Payments made 

#### Development Checklist:

- [ ] Create eLicensing endpoint to query invoice payments
- [ ] Create API endpoint to query invoice payments for an obligation
- [ ] Create UI to display monetary payments
- [ ] Integarte UI with BE to display monetary payments
- [ ] Meets the DOD

#### Definition of Ready (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

#### Definition of Done (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By/Blocking
Blocked by #130 #18 and eLicensing end points

#### Notes:

-

**Comment by fviduya** [comment]
I did some additional testing for this which can be found [here](https://buttoninc-my.sharepoint.com/:w:/g/personal/francesca_button_is/EX8HNZOxV9lCrN7wtLbENRUBZKyJAuYikswiAaPQ-lr8fw?e=Jxirvb&nav=eyJoIjoiODc4NDI2ODg4In0%3D):

- [x] The monetary payments paid by the operator are visible in the Monetary Payments Made section 
- [x] When operator makes multiple payments for the operation, the payments are visible in the Monetary Payments Made section of the specific obligation record

**Comment by vesselak** [comment]
Reviewed the testing results and accepting ticket. thanks
